### DISCUSSION TITLE
Feedback Full Screen

### DISCUSSION THREAD

**Discussion by Sineos** [teeminus/NoTouchScreenFirmware#27] [General]
Nice one. Works for my E3 V3 display. 
The step above would be Braille 😏

**Comment by teeminus** [comment]
I think thats a different kind of touch screen ;)

Good to know that it works on other screens too :)

**Comment by emtrax-ltd** [comment]
works very reliably with the emulated st7920 - delayed pwm dimming and color customization are also working fine.

scrren: btt tft35 e3 v3.0
board: skr mini e3 v1.2 
klipper: emulated st7920

![fullscreen](https://user-images.githubusercontent.com/60063640/111771551-21ef1080-88ac-11eb-8bf4-356d969cf457.png)

**Comment by mikey63** [comment]
wanted to chime in and say this is working just fine on the BTT TFT50 as well, other than the direction on the rotary encoder is swapped.  I looked in the encoder.h and .c files, but it's not obvious which line affects that.

also, I think I'm going to try and turn the brightness down to 75% or so.  it's really bright!

thanks for making this!

**Reply by teeminus** [reply]
The rotary encoder is not decoded by the touchscreen but by the firmware of the 3d printer. If you are using Klipper, you can just swap the two encoder pins in your config file.

**Reply by mikey63** [reply]
That was it!!  works perfectly now, thanks!   I also recompiled with 75% brightness, and all is well.
